src/LessonOOPstarter/Initial/BankAccount/BankAccount.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BankAccount
+{
+    class BankAccount
```

Comment:
Inca nu as adauga clasa in sine in proietul initial. Ar fi frumos sa o facem impreuna cu ei

Reply:
O sa o sterg atunci